AutoMaxLair.py

```diff
@@ -386,17 +384,19 @@ def backpacker(inst) -> str:
 
     inst.log("Reading the backpacker's items.")
 
-    f = open('itemsList.txt', 'a', encoding='utf8')
     items = []
```

Comment:
here is a real change. From all the run I made, I found the list of 24 items I gave. Not a single item not present in that list

Reply:
Ok great. I have been collecting item names as well so we can compare.